src/porepy/numerics/time_step_control.py

```diff
@@ -666,7 +666,7 @@ def is_schedule_in_simulated_times(
         return schedule.size == np.sum(in1d)
 
     # I/O
-    def write_time_information(self, path: Optional[Path] = None) -> None:
+    def write_time_information(self, path: Optional[Path | str] = None) -> None:
```

Comment:
@keileg, can you comment on this one? Are we trying to enforce Path variables?

Reply:
@IvarStefansson this change is motivated by this line:
https://github.com/pmgbergen/porepy/blob/b92d5f98b1f313eaa1b96463c267d16c2749acfa/tests/unit/test_time_step_control.py#L659

Reply:
I see. I still want Eirik's opinion, having heard him mention this recently.

Reply:
File name should be represented as Paths. I will take care of this one.